cite/bibliography.bib

```diff
@@ -481,6 +601,13 @@ @article{RN901
   publisher = {Oxford University Press ({OUP})}
 }
 
+@techreport{Hurst2021,
+  author = {Hurst, Thomas P. and O'Leary, Cecilia A. and Rohan, Sean K. and Siddon, Elizabeth C. and Thorson, James T. and Vollenweider, Johanna J.},
+  doi = {10.25923/p1yd-0793},
+  title = {{Inventory, management uses, and recommendations for fish and crab condition information from the 2021 AFSC Condition Congress. AFSC Processed Rep. 2021-04, 39 p. Alaska Fish. Sci. Cent., NOAA, Nat. Mar. Fish. Serv., 7600 Sand Point Way NE, Seattle, WA 981}},
```

Comment:
Parts of this will need to be split out into different bibtext elements

Reply:
Is this related to a specific reference or all of them?

Reply:
Specifically to line 607: `title = {{Inventory, management uses, and recommendations for fish and crab condition information from the 2021 AFSC Condition Congress. AFSC Processed Rep. 2021-04, 39 p. Alaska Fish. Sci. Cent., NOAA, Nat. Mar. Fish. Serv., 7600 Sand Point Way NE, Seattle, WA 981}},`

Reply:
basically the title has a bunch of other stuff in it (like pages, report number), so line 607 would be rewritten as something like this: 
`title = {{Inventory, management uses, and recommendations for fish and crab condition information from the 2021 AFSC Condition Congress}},`
`number = {AFSC Processed Rep. 2021-04}, `
`pages = {39},`
`institution = {{NOAA Fisheries Alaska Fisheries Science Center}},`